### PR TITLE
added env for local development with backend running locally

### PR DESCRIPTION
**Pull request by xavierbloemen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
   },
   "scripts": {
     "start": "PORT=3001 react-scripts start",
+    "start:local": "env-cmd -f .env.local react-scripts start",
```
Comment (Jeffrey-Zutt) [inline package.json:36]: There is no .env.local

File: .env.development [hunk 2]
```diff
@@ -2,7 +2,7 @@
 REACT_APP_GATEWAY=https://acc.api.top.amsterdam.nl/api/v1/
 
 # To login using grip:
-REACT_APP_AUTH_URL="https://auth.grip-on-it.com/v2/rjsfm52t/oidc/idp/authorize?response_type=code&scope=openid&client_id=d3d664c7-bb33-4bf0-b7c9-b8bdf1199b76&redirect_uri=http%3A%2F%2Flocalhost%3A3001%2Fauthentication%2Fcallback"
+REACT_APP_AUTH_URL="/authentication/callback?code=1234567890abcdefghijkl"
```
Comment (Jeffrey-Zutt) [inline .env.development:5]: If this is the only change needed.
I'd advice to create a .env locally and dont check it it in.

The .env should contain only this line.

Comment (xavierbloemen) [reply .env.development:5]: excuus pas ik weer terug aan

File: package.json (hunk 1 above)
Comment (xavierbloemen) [reply package.json:36]: ah de .gitignore negeerde die file name. pas het aan

File: .env.development (hunk 2 above)
Comment (xavierbloemen) [reply .env.development:5]: was iets te snel met de commit

**Review by Jeffrey-Zutt — APPROVED** [review]

**Pull request merged**